galpy/potential/DissipativeForce.py

```diff
@@ -189,20 +190,19 @@ def _isDissipative(obj):
     - 2023-05-29 - Adjusted to also take planarDissipativeForces into account - Bovy (UofT)
 
     """
-    from .planarDissipativeForce import planarDissipativeForce
-    from .Potential import flatten
+    from .Potential import PotentialError, flatten
 
     obj = flatten(obj)
     isList = isinstance(obj, list)
     if isList:
-        isCons = [
-            not isinstance(p, DissipativeForce)
-            and not isinstance(p, planarDissipativeForce)
-            for p in obj
-        ]
+        isCons = [not _isDissipative(p) for p in obj]
```

Comment:
Apparently this leads to a few test failures:

https://github.com/jobovy/galpy/actions/runs/11911054548/job/33202046778

I think these can all be fixed by changing the test to check for ``AttributeError`` rather than ``PotentialError``; it's fine for these to just raise an ``AttributeError``.